lib/builder.js

```diff
@@ -122,7 +123,7 @@ function writeOutputFile(opts, output, basePath) {
   });
 }
 
-exports.inlineSourceMap = function(output) {
+var inlineSourceMaps = exports.inlineSourceMap = function(output) {
```

Comment:
Plurals were tricksy - got bitten in both directions. We don't control `output`, but we can change `opts` (queue breaking grunt users again).